code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1154,12 +1154,14 @@
 
 
 		switch(hit_area)
-			if("head")	//Harder to score a stun but if you do it lasts a bit longer
+			if("head")	//Causes dizzness, brain damage and forces the target to drop their items
 				if(H.stat == CONSCIOUS && armor_block < 50)
-					if(prob(I.force))
-						H.visible_message("<span class='danger'>[H] has been knocked unconscious!</span>", \
-										"<span class='userdanger'>[H] has been knocked unconscious!</span>")
-						H.apply_effect(20, PARALYZE, armor_block)
+					if(prob(min(I.force, 25)))
```

Comment:
so it's ok if an item with force set to 1 gives you a concussion?sounds weird as fuck imo